guides/common/modules/con_content-views-in-projectname.adoc

```diff
@@ -0,0 +1,46 @@
+[id="Content-Views-in-{ProjectNameID}_{context}"]
+= Content Views in {ProjectName}
+
+A _content view_ defines a deliberately curated subset of content that your hosts can access.
+Each content view creates a set of repositories across each environment.
+The {ProjectServer} stores and manages these repositories.
+You can use content views to define which software versions a particular environment or {SmartProxyServer} uses.
+For example:
+
+* You can create a content view with older package versions for a Production environment and another content view with newer package versions for a Development environment.
+* You can create a content view with a package repository required by an operating system and another content view with a package repository required by an application.
+
+You can also combine content views to create a _composite content view_ for ease of management.
```

Comment:
This could be improved. Intro to a list should be a full sentence. This will also help us not repeat information in each bullet. I would also bulletify the last line:

```suggestion
For example, you can create content views in the following ways:

* A content view with older package versions for a production environment and another content view with newer package versions for a Development environment.
* A content view with a package repository required by an operating system and another content view with a package repository required by an application.
* A _composite content view_ that combines content views for easier management.
```